templates/repo/header.tmpl

```diff
@@ -6,7 +6,9 @@
 				{{if .RelAvatarLink}}
 					<img class="ui avatar image" src="{{.RelAvatarLink}}">
 				{{else}}
-					{{template "repo/header_icon" .}}
+					<div class="repo-header-icon">
+						{{template "repo/header_icon" .}}
+					</div>
```

Comment:
I meant you should to move the `<div>` inside `repo/header_icon`.

Reply:
Oh, sorry.